File: 6-kyu/Multiples of 3 and 5/Multiples3n5.js
// Kata found at: https://www.codewars.com/kata/514b92a657cdc65150000006

function solution(number){
  
  var count = 0;
  for(var i = 3; i < number; i++)  {
  
    if(i % 3 == 0 || i % 5 == 0)  {count += i;}
  
  }
  return count;
  
}

File: 6-kyu/Multiples of 3 and 5/Multiples3n5.cpp
// Kata found at: https://www.codewars.com/kata/514b92a657cdc65150000006

int solution(int number) 
{
  int count = 0;
  for(int  i = 1; i < number; i++)  {
  
    if(i % 3 == 0 || i % 5 == 0)  {
    
      count += i;
    
    }
  
  }
  return count;
}
